src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
I'm not really suggesting anything specific just providing some info. "Address" is a heavily abused term in the rippled code base. Specifically, a `RippleAddress` is a "kitchen sink" type object which can hold almost anything. It can hold a "node id", AccountID, public key, private key, validator public key, validator private key, and more. Which breaks the type-safety of the C++ language and leads to all kinds of unnecessary ugliness.